game/src/main/scala/io/aigar/controller/AdminResponse.scala

```diff
@@ -2,4 +2,18 @@ package io.aigar.controller.response
 
 case class AdminQuery(
   administrator_password: String
+ )
+
+case class SetRankedDurationQuery(
+  administrator_password: String,
```

Comment:
Could we update [`the API docs`](https://github.com/DrPandemic/aigar.io/blob/master/API.md#request-structure) to match the *underscore_casing* used instead of *camelCasing*?